index.html

```diff
@@ -1,199 +1,315 @@
+                        <li>Ruby</li>
+                        <li>HMTL</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+            </div>
 
-                    <div>
-                        <div class="square-card">
-                            <div class="language-img">
-                            <img src="images-2/third-icon.svg" alt="skills">
-                            </div>
-                        <h3>skills</h3>
-                        <ul class="my-languages">
-                            <li>Database Management</li>
-                            <li>Version Controle</li>
-                            <li>CLI</li>
-                            <li>WEb Development</li>
-                            <li>API Design</li>
-                        </ul>
-                        </div>
+            <div>
+                <div class="square-card">
+                    <div class="language-img">
+                        <img src="images-2/second-icon.svg" alt="Framewokrs">
                     </div>
+                    <h3>frameworks</h3>
+                    <ul class="my-languages">
+                        <li>React.js</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+            </div>
 
+            <div>
+                <div class="square-card last-square-card">
+                    <div class="language-img">
+                        <img src="images-2/third-icon.svg" alt="skills">
                     </div>
-                    
-                    
-                </section>
-        </article>
-        <footer>
-            <div class="data">
-            <p class="comment">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-            <form  action="https://formspree.io/f/xqkovozr" method="POST" class="form">
-              <input class="input_name" type="text" placeholder="Full name" name="name" maxlength="30" required/>
-              <input class="input_email" type="email" placeholder="Email address" name="email" required="email"/>
-              <textarea class="input_msg" placeholder="Hey I'd like to connect to..." name="message" rows="8" maxlength="500" required ></textarea>
-              <div class="Get-in-touch-button"><button class="interaction-button" type="submit">Get in touch</button></div>
+                    <h3>skills</h3>
+                    <ul class="my-languages">
+                        <li>Database Management</li>
+                        <li>Version Controle</li>
+                        <li>CLI</li>
+                        <li>WEb Development</li>
+                        <li>API Design</li>
+                    </ul>
+                </div>
             </div>
+
+        </div>
+
+</div>
+    </section>
+    </article>
```

Comment:
- [ ] According to your previouse reviewer there is a closing article tag in your code without a corresponding opening tag. It seems to be unnecessary and may cause confusion for other developers reviewing your code. I recommend removing the closing article tag since there is no associated opening tag.

Keeping your code clean and free of unnecessary tags will make it easier to understand and maintain. Please consider removing the closing article tag to ensure the code's integrity. Great attention to detail! 👍